articles/azure-sql/database/database-export.md

```diff
@@ -98,6 +98,9 @@ while ($exportStatus.Status -eq "InProgress")
 [Console]::WriteLine("")
 $exportStatus
 ```
+## Cancel the export request
+
+Use the procedure in the blog [How to cancel Azure SQL Database Import or Export operation](https://techcommunity.microsoft.com/t5/azure-database-support-blog/how-to-cancel-azure-sql-database-import-or-export-operation/ba-p/1935001) to cancel an ongoing import or export operation on Azure SQL Database. 
```

Comment:
We don't generally point to blogs from docs.microsoft.com. General rule is, that blogs can point to documentation, but documentation does not point to blogs, because blogs and their links can change anytime. Instead of pointing to the blog from here, could you please document a simple example of canceling an operation using REST API / PowerShell. You can then link to our REST API and Powershell documentation from here.

Reply:
## Cancel the export request
Use the [Database Operations - Cancel API] (https://docs.microsoft.com/en-us/rest/api/sql/databaseoperations/cancel)
or the Powershell [Stop-AzSqlDatabaseActivity command] (https://docs.microsoft.com/en-us/powershell/module/az.sql/Stop-AzSqlDatabaseActivity?view=azps-5.5.0)

Example:-
Stop-AzSqlDatabaseActivity -ResourceGroupName $ResourceGroupName -ServerName $ServerName -DatabaseName $DatabaseName -OperationId $Operation.OperationId

Reply:
## Cancel the export request
Use the [Database Operations - Cancel API] (https://docs.microsoft.com/en-us/rest/api/sql/databaseoperations/cancel)
or the Powershell [Stop-AzSqlDatabaseActivity command] (https://docs.microsoft.com/en-us/powershell/module/az.sql/Stop-AzSqlDatabaseActivity?view=azps-5.5.0)

Example:-
Stop-AzSqlDatabaseActivity -ResourceGroupName $ResourceGroupName -ServerName $ServerName -DatabaseName $DatabaseName -OperationId $Operation.OperationId